CMakeLists.txt

```diff
@@ -523,6 +523,13 @@ pika_option(
   ADVANCED
 )
 
+pika_option(
+  PIKA_WITH_SWAP_CONTEXT_EMULATION BOOL
+  "Emulate SwapContext API for coroutines (Windows only, default: OFF)" OFF
+  CATEGORY "Thread Manager"
+  ADVANCED
```

Comment:
is omitting `MODULE COROUTINE` intentional?

Reply:
Sounds good to remove it!

Reply:
I removed the `MODULE` option completely from `pika_option` along with the `Modules` category. Does this look better now?